Makefile

```diff
@@ -176,14 +176,14 @@ launch: install check-kubectl
 	kubectl wait --for=condition=ready pod -l app=pachd --timeout=5m
 	@echo "pachd launch took $$(($$(date +%s) - $(STARTTIME))) seconds"
 
-launch-dev: check-kubectl check-kubectl-connection install
+launch-dev: check-kubectl check-kubectl-connection
 	$(eval STARTTIME := $(shell date +%s))
-	helm install pachyderm etc/helm/pachyderm -f etc/helm/examples/local-dev-values.yaml
+	helm install pachyderm etc/helm/pachyderm -f etc/helm/examples/local-dev-values.yaml --set $(LAUNCH_DEV_ARGS)
```

Comment:
I'm not sure how other people deal with this and I didn't want to step on someone else's workflow (i.e. they may do multiple deployments and don't want them both set to `dev`), but it might be worth having a team discussion on this once things settle down.